src/components/TestimonialsSection.vue

```diff
@@ -57,4 +57,22 @@ export default {
 };
 </script>
 
-<style scoped></style>
+<style scoped>
+/* ------------------------------------------------------------------
+/* g. Testimonials
+/* ------------------------------------------------------------------ */
+
+#testimonials {
```

Comment:
Move this style to the public folder. once you move it, change the background image path to "/images/testimonials-bg.jpg" and the image should be in this path in the public folder.